source/_integrations/shelly.markdown

```diff
@@ -331,7 +331,9 @@ The integration supports the following virtual components:
 
 ## Scripts (generation 2 and 3)
 
-For each device script, the integration creates a `switch` entity that allows you to control the script. These entities are disabled by default.
+For each device script, the integration creates a `switch` entity that allows you to control the script and an `event` entity. These entities are disabled by default.
+
+The `event` entity shows event that are emitted by the script using [Shelly.emitEvent()](https://shelly-api-docs.shelly.cloud/gen2/Scripts/ShellyScriptLanguageFeatures#shellyemitevent). After changing a script, it's required to manually reload the device before new event types show up.
```

Comment:
It would be nice to add a short example script (maybe something similar to what is used as MOCK in the core PR) in addition to the link to Shelly docs.